drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1453,8 +1474,42 @@ function DragAndDropBlock(runtime, element, configuration) {
                     cancelAnimationFrame(raf_id);
                 }
                 raf_id = requestAnimationFrame(function() {
-                    var dx = pageX(evt) - drag_origin.x;
-                    var dy = pageY(evt) - drag_origin.y;
+                    var x = pageX(evt);
+                    var y = pageY(evt);
+
+                    var $target = $container.find('.target');
+                    var $target_rect = $target.get(0).getBoundingClientRect();
+                    var top = $target_rect.top + window.scrollY;
+                    var bottom = $target_rect.bottom + window.scrollY;
+                    var left = $target_rect.left + window.scrollX - 50;
```

Comment:
Nit: can we make the hardcoded `50` value a variable? It gets repeated several times.

Reply:
Done.